.eslintrc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   extends: [
     "plugin:@typescript-eslint/recommended", // Uses the recommended rules from the @typescript-eslint/eslint-plugin
     "prettier/@typescript-eslint", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
-    "plugin:prettier/recommended", // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+    "prettier", // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
```

Comment:
To samo nie wystarczy, żeby priettier działał z naszymi ustawieniami. Ale możesz to zostawić, ja mam już poprawkę w swoim PR.

Reply:
nie ale wystarczy żebym nie miał wszystkiego (niesłusznie) na czerwono w IDE 😄